src/common/mmo.h

```diff
@@ -838,6 +838,13 @@ enum e_pc_reg_loading {
 	PRL_ALL = 0xFF,
 };
 
+typedef enum PARTY_MEMBER_WITHDRAW {
+	PARTY_MEMBER_WITHDRAW_LEAVE,	  ///< /leave
+	PARTY_MEMBER_WITHDRAW_EXPEL,	  ///< Kicked
+	PARTY_MEMBER_WITHDRAW_CANT_LEAVE, ///< TODO: Cannot /leave
```

Comment:
Could that be the correct return for clif_parse_LeaveParty when you are on a party locked map?
But that might be PACKETVER dependent.